cmd/pitr/pxc/pxc.go

```diff
@@ -105,6 +107,25 @@ func (p *PXC) GetBinLogList() ([]Binlog, error) {
 	return binlogs, nil
 }
 
+// GetBinLogList return binary log files list
+func (p *PXC) GetBinLogNamesList() ([]string, error) {
+	rows, err := p.db.Query("SHOW BINARY LOGS")
```

Comment:
We should do defer rows.Close() to release connection in case of error

Reply:
we have Close() method in defer

Reply:
As I understand it closes whole db. I'm talking about closing one of connection from pool.

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/761/commits/76be207197315bc5a9af18272da5705c1660b9e4